src/acp/rule.ts

```diff
@@ -201,9 +213,9 @@ export function setAnyOfRuleUrl<P extends Policy | ResourcePolicy>(
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Get the [[Rule]]'s accepted by the given [[Policy]]
+ * Get the [[Rule]]s accepted by the given [[Policy]]
```

Comment:
`getAllOfRuleUrlAll` uses "Get the "All Of" [[Rule]]s...". Should this use "Get the "Any Of" [[Rule]]s..." instead of "accepted" ?

Reply:
Thanks for checking, I really lost track of keeping everything consistent at some point... 2cc92033cba5d2642e628f19235cbc6c712274ea